packaging/make_deb.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/bash
+
+DISTRIBUTION="_$1"
+echo $DISTRIBUTION
+
+# Compress the changelog, strip the binaries
+cp changelog.Debian calculix-precice_2.17-1_amd64/usr/share/doc/calculix-precice/changelog.Debian
+# Options : -f for removing file if it was there, -n for no time stamp
+gzip -9 calculix-precice_2.17-1_amd64/usr/share/doc/calculix-precice/changelog.Debian -f -n
+strip --strip-unneeded calculix-precice_2.17-1_amd64/usr/bin/ccx_preCICE
+
+#Compile and compress the manual
+
+pandoc manpage.md -s -t man -o ccx_preCICE.1
+mkdir -p calculix-precice_2.17-1_amd64/usr/share/man/man1
+gzip -9 -n -f ccx_preCICE.1
+chmod 644 ccx_preCICE.1.gz
+mv ccx_preCICE.1.gz calculix-precice_2.17-1_amd64/usr/share/man/man1
+
+# Copy to a folder with appropriate postfix
+
+cp -r calculix-precice_2.17-1_amd64/ "calculix-precice_2.17-1_amd64$DISTRIBUTION"
```

Comment:
I don't really like that we have the versions hardcoded everywhere. But I also understand that it is complicated to pass this information everywhere needed.

Maybe add some documentation in the README on where one needs to update the version and what the versioning scheme is?

Reply:
I had added a file "README for maintainers" at some point but it seems I lost it / forgot to add it to Git. I'll rewrite that.